ui/raidboss/data/04-sb/trial/shinryu.js

```diff
@@ -0,0 +1,549 @@
+        'Icicle Impact': 'アイシクルインパクト',
+        'Judgment Bolt': '裁きの雷',
+        'Levinbolt': '稲妻',
+        'Meteor Impact': 'メテオインパクト',
+        'Protostar': 'プロトスター',
+        'Spikesicle': 'アイシクルスパイク',
+        'Summon Icicle': 'サモン・アイシクル',
+        'Super Cyclone': 'スーパーサイクロン',
+        'TAP BUTTON OR ELSE': 'TAP BUTTON OR ELSE',
+        'Tail Slap': 'テールスラップ',
+        'Tidal Wave': 'タイダルウェイブ',
+      },
+      '~effectNames': {
+        'Affixed': 'しがみつき',
+        'Burning Chains': '炎の鎖',
+        'Burns': '火傷',
+        'Electrocution': '感電',
+        'Fetters': '拘束',
+        'Fire Resistance Up': '火属性耐性向上',
+        'Lightning Resistance Down II': '雷属性耐性低下［強］',
+        'Magic Vulnerability Up': '被魔法ダメージ増加',
+        'Physical Vulnerability Up': '被物理ダメージ増加',
+        'Stun': 'スタン',
+        'Thin Ice': '氷床',
+      },
+    },
+    {
+      'locale': 'cn',
+      'missingTranslations': true,
+      'replaceSync': {
+        'Cocoon': '光茧',
+        'Left Wing': '左翼',
+        'Right Wing': '右翼',
+        'Shinryu': '神龙',
+        'Tail': '龙尾',
+      },
+      'replaceText': {
+        '--adds spawn--': '--adds spawn--',
+        'Aerial Blast': '大气爆发',
+        'Akh Morn': '死亡轮回',
+        'Akh Rhai': '天光轮回',
+        'Elemental Attack': '大气爆发/制裁之雷/钻石星尘/地狱之火炎/大地之怒/巨浪',
+        'Dark Matter': '暗物质',
+        'Diamond Dust': '钻石星尘',
+        'Dragonfist': '龙掌',
+        'Earth Breath': '大地吐息',
+        'Earthen Fury': '大地之怒',
+        'Gyre Charge': '螺旋冲锋',
+        'Hellfire': '地狱之火炎',
+        'Hypernova': '超新星',
+        'Ice Storm': '吹雪',
+        'Icicle Impact': '冰柱冲击',
+        'Judgment Bolt': '制裁之雷',
+        'Levinbolt': '闪电',
+        'Meteor Impact': '陨石冲击',
+        'Phase': '阶段',
+        'Protostar': '原恒星',
+        'Spikesicle': '冰柱突刺',
+        'Summon Icicle': '召唤冰柱',
+        'Super Cyclone': '超级气旋',
+        'TAP BUTTON OR ELSE': 'XJB按',
+        'Tail Slap': '尾部猛击',
+        'Tidal Wave': '巨浪',
+      },
+      '~effectNames': {
+        'Affixed': '紧抱',
+        'Burning Chains': '火焰链',
+        'Burns': '火伤',
+        'Electrocution': '感电',
+        'Fetters': '拘束',
+        'Fire Resistance Up': '火属性耐性提升',
+        'Lightning Resistance Down II': '雷属性耐性大幅降低',
+        'Magic Vulnerability Up': '魔法受伤加重',
+        'Physical Vulnerability Up': '物理受伤加重',
+        'Stun': '眩晕',
+        'Thin Ice': '冰面',
+      },
+    },
+    {
+      'locale': 'ko',
+      'missingTranslations': true,
+      'replaceSync': {
+        'Cocoon': '빛의 고치',
+        'Left Wing': '왼쪽 날개',
+        'Right Wing': '오른쪽 날개',
+        'Shinryu': '신룡',
+        'Tail(?! )': '신룡의 꼬리',
+      },
+      'replaceText': {
+        '--adds spawn--': '--adds spawn--',
+        'Aerial Blast': '대기 폭발',
+        'Akh Morn': '아크 몬',
+        'Akh Rhai': '아크 라이',
+        'Dark Matter': '암흑물질',
+        'Diamond Dust': '다이아몬드 더스트',
+        'Dragonfist': '용의 손바닥',
+        'Dragonflight': '용의 비행',
+        'Elemental Attack': 'Elemental Attack',
```

Comment:
`'Elemental Attack': '원소 공격',`

Reply:
Thanks! Do note, it is possible we may change the English on this one, depending on what other reviewers have to say. For reference, this mechanic is one of six possible attacks:
```
Aerial Blast
Judgment Bolt
Diamond Dust
Hellfire
Earthen Fury
Tidal Wave
```
We can't list all of them in the timeline, so I shortened it here. (You can see that because the CN translations for these attacks are all four characters long, it was easy to just put all six into that replacement.)

Reply:
I saw that in timeline file, and I think this translation is okay with it, better than just listing all the skill names.